lib/google_contacts_api/result_set.rb

```diff
@@ -7,34 +7,36 @@ class ResultSet
     include Enumerable
     attr_reader :api
     attr_accessor :total_results, :start_index, :items_per_page, :parsed
-    
+
     # Initialize a new ResultSet from the response, with the given
     # GoogleContacts::Api object if specified.
     def initialize(response_body, api = nil)
       @api = api
       @parsed = Hashie::Mash.new(JSON.parse(response_body))
-      @total_results = @parsed.feed["openSearch$totalResults"]["$t"].to_i
-      @start_index = @parsed.feed["openSearch$startIndex"]["$t"].to_i
-      @items_per_page = @parsed.feed["openSearch$itemsPerPage"]["$t"].to_i
+      unless @parsed.feed.nil?
```

Comment:
When getting a single contact via "contacts/default/full/#{params[:contact_id]}" the returned JSON does not have a "feed" field.